src/gtest/test_mempool.cpp

```diff
@@ -443,8 +443,12 @@ class CNodeExt : public CNode
     }
 };
 
+extern std::unique_ptr<CConnman> connman;
+
 TEST(ProcessMempoolMsgTest, TxesInMempoolAreRelayed)
 {
+    connman.reset();
```

Comment:
Maybe this can made a one-liner:
```suggestion
    connman.reset(new CConnman());
```
if you agree, please consider also three occurrences in `DoS_tests.cpp`.

Reply:
Yes, I also like this more compact notation. Changed as requested.

Reply:
If there is no rethinking let's remove the old commented code for those four occurrences, thanks.

Reply:
You are right, forgot to remove the commented lines. Fixed.